ui/ui-components/pages/model/[modelId]/source/[sourceId]/edit.tsx

```diff
@@ -49,6 +57,113 @@ interface Props {
   preview: SourcePreviewMethodResponseRow[];
 }
 
+interface TypeaheadProps {
+  control: Control<FormData, object>;
+  opt: ConnectionOptionsOption;
+  connectionOptions: SourceOptionsMethodResponse;
+  loading: boolean;
+  loadingOptions: boolean;
+  source: Pick<Models.SourceType, "name" | "options">;
+  updateOption: (optKey: string, optValue: string) => void;
+  register: UseFormRegister<FormData>;
+}
+
+const TypeaheadDeterminer = ({
```

Comment:
`TypeaheadProps` and `TypeaheadDeterminer` don't quite feel like the right name for these... `Typeahead` is _one_ of the things this could return, but really it's rendering any type of property

Reply:
Yeah.  Bad copy/paste there.